docs/release-notes/8.9.asciidoc

```diff
@@ -1,6 +1,31 @@
 [[release-notes-header-8.9.0]]
 == 8.9
 
+[discrete]
+[[release-notes-8.9.1]]
+=== 8.9.1
+
+[discrete]
+[[breaking-changes-8.9.1]]
+==== Breaking changes
+
+There are no breaking changes in 8.9.1.
+
+[discrete]
+[[enhancements-8.9.1]]
+==== Enhancements
+* Event correlation rules now support negative correlation. 
```

Comment:
@paulewing let me know if you'd like for this summary to be expanded and linked to a Security PR - thanks!